lib/tls.js

```diff
@@ -109,8 +109,8 @@ function convertProtocols(protocols) {
     return p + 1 + len;
   }, 0));
 
-  var offset = 0;
-  for (var i = 0, c = protocols.length; i < c; i++) {
+  let offset = 0;
+  for (let i = 0, c = protocols.length; i < c; i++) {
```

Comment:
nit: offset can be moved inside loop as follows:
```suggestion
  for (let offset = 0, i = 0, c = protocols.length; i < c; i++) {
```